docs/smart-contract-backend/scb.md

```diff
@@ -69,6 +69,38 @@ The node client is a process that talks to a Cardano node. It can submit transac
 2. Ask for all unspent outputs at an address (see 1.2.1.5)
 3. Ask for the current slot number (see 1.2.1.1)
 
+We communicate with the node client using a REST API, and the listener will establish a web sockets connection and be updated regularly with accepted transactions.
+
+The mock will implement the following endpoints (which will also be exposed by the real node):
+
+```
+--| Notations
+    <-   Client sends to Server.
+    ->   Server sends to Client.
+    <d>  Datum `d` is JSON encoded.
+
+data Update = Rollback Slot <Block>
+            | NewBlock Slot <Block>
+
+-- API
+GET /utxo/:addr
+-> <[(TxOutRef, TxOut)]>
+
+GET /currentSlot
+-> <Slot>
+
+GET /chain/:slot -- Web sockets stream.
```

Comment:
I think this should be `block`, as the slot is more of a concern for the node.

Reply:
I think, if we provide the slot number on the web socket interface we will not need this call. Am I correct @j-mueller ?